ndk/src/main/java/com/bugsnag/android/ndk/NativeBridge.java

```diff
@@ -54,7 +54,9 @@ public static native void addBreadcrumb(String name, String type, String timesta
 
     public static native void removeMetadata(String tab, String key);
 
-    public static native void startedSession(String sessionID, String key);
+    public static native void startedSession(String sessionID, String key, int handledCount);
```

Comment:
I don't think that we can always be sure that unhandled count will be zero, so do we also need to pass that through here

Reply:
Passing through the `unhandledCount` will need to be done when we implement ANR detection. Doing this would add an additional field to the `bugsnag_report` struct, so would require a migration - therefore we'll wait until the ANR project to implement this.